src/plugins/visualize/public/application/components/visualize_listing.tsx

```diff
@@ -123,7 +123,9 @@ export const VisualizeListing = () => {
         .then(({ total, hits }: { total: number; hits: object[] }) => ({
           total,
           hits: hits.filter(
-            (result: any) => isLabsEnabled || result.type?.stage !== 'experimental'
+            (result: any) =>
+              isLabsEnabled ||
+              (result.type?.stage !== 'experimental' && result.stage !== 'experimental')
```

Comment:
This is because depending on the type of visualization (using the 'visualization' saved object or 'visBuilder' visualization) the location of the experimental property changes. I'd add a comment but honestly why they are in two different locations is beyond me so a comment would essentially describe this as you see it in the code. We dont have other aliased visualization to know for sure if its specific to aliased visualizations or just VisBuilder